src/Akeneo/Connectivity/Connection/back/Domain/Audit/Model/Read/ErrorCountPerConnection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Audit\Model\Read;
+
+/**
+ * @author Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+final class ErrorCountPerConnection
+{
+    /** @var ErrorCount[] */
+    private $errorCounts;
+
+    /**
+     * @param ErrorCount[] $errorCounts
+     */
+    public function __construct(array $errorCounts)
+    {
+        foreach($errorCounts as $errorCount) {
+            if (false === $errorCount instanceof ErrorCount) {
+                throw new \InvalidArgumentException('One of the given element is not an ErrorCount object.');
+            }
+
+            $this->errorCounts[] = $errorCount;
+        }
+    }
+
+    public function normalize(): array
+    {
+        $errorCountPerConnection = [];
+
+        foreach ($this->errorCounts as $errorCount) {
+            $errorCountPerConnection[] = $errorCount->normalize();
```

Comment:
```suggestion
            $data = $errorCount->normalize();
            $errorCountPerConnection[$data['connection_code']] = (int) $data['count'];
```
We defined a format like this for the internal API response:
```json
{
  "code_1": 3,
  "code_2": 9
}
```